src/utils/userManager.js

```diff
@@ -5,13 +5,13 @@ const host = `${window.location.protocol}//${window.location.hostname}${
   window.location.port ? `:${window.location.port}` : ""
 }`;
 
-const userManagerConfig = {
+const userManagerConfig = ({ reactAppAuthenticationUrl }) => ({
```

Comment:
❓ I'm struggling to see where `reactAppAuthenticationUrl` is set 🤔 I'm sure its just me but I can't see it 😅 or do we not need it?

Reply:
Yeah I had the same realisation when doing this 😅 
It's actually not yet used - only in `src/app/store.js` where I've left a TODO:

```
// TODO - not used but keeping this in preparation for using
// src/components/Editor/ImageUploadButton/ImageUploadButton.jsx
```

Reply:
that's mad that's the only use of that variable 😅